x-pack/plugins/siem/server/lib/detection_engine/signals/signal_rule_alert_type.ts

```diff
@@ -155,13 +156,18 @@ export const signalRulesAlertType = ({
             await ruleStatusService.error(errorMessage);
           }
 
-          const anomalyResults = await findMlSignals(
-            machineLearningJobId,
+          const anomalyResults = await findMlSignals({
+            ml,
+            callCluster: scopedMlCallCluster,
+            // This is needed to satisfy the ML Services API, but can be empty as it is
+            // currently unused by the mlSearch function.
+            request: ({} as unknown) as KibanaRequest,
```

Comment:
As mentioned in the above comment, this has no effect right now other than to appease typescript; the request is unused in the `mlSearch` function, and [this issue](https://github.com/elastic/kibana/issues/64588#issuecomment-622010446) will need to be resolved before said request can be leveraged.